vector_double.hpp

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
some weird spacing in this file, can you clean it up just a bit? otherwise looks perfect